public/Aframe/samples/number_lock.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>CoronaEscape</title>
+        <meta name="description" content="CoronaEscape" />
+        <script src="https://aframe.io/releases/1.1.0/aframe.min.js"></script>
+        <script src="aframe-super-keyboard.min.js"></script>
+        <script src="keypad.js"></script>
+    </head>
+    <body>
+        <a-scene
+            cursor="rayOrigin:mouse"
+            background="color: #ECECEC"
+            raycaster="objects: .link"
+        >
+            <a-box
+                static-body="shape: auto"
+                position="0 -0.52 0"
+                rotation="90 0 0"
+                width="20"
+                height="10"
+                depth="1"
+                floor
+            ></a-box>
+            <a-box
+                static-body="shape: auto"
+                position="0 0.5 -4.5"
+                width="10"
+                height="5"
+                depth="1"
+                floor
+            ></a-box>
+            <a-box
+                static-body="shape: auto"
+                position="0 0.5 4.5"
+                width="10"
+                height="5"
+                depth="1"
+                floor
+            ></a-box>
+            <a-box
+                static-body="shape: auto"
+                position="-5.1 0.5 0"
+                rotation="0 90 0"
+                width="10"
+                height="5"
+                depth="1"
+                floor
+            ></a-box>
+            <a-box
+                static-body="shape: auto"
+                position="4.5 0.5 0"
+                rotation="0 90 0"
+                width="10"
+                height="5"
+                depth="1"
+                floor
+            ></a-box>
+            <a-box
+                static-body="shape: auto"
+                position="0 4 0"
+                rotation="90 0 0"
+                width="20"
+                height="10"
+                depth="1"
+                color="white"
+                floor
+            ></a-box>
+            <!-- <a-entity id="keypad-popup-container" visible="true"> -->
+            <a-entity
+                id="keyboard-display"
+                position="0 1.2 -3.9"
+                rotation="0 0 0"
+                scale="3 3 3"
+                super-keyboard="hand: #mouseCursor; imagePath:./; multipleInputs:true; filters: numbers; model: numpad; align: center; maxLength: 4; label: Enter Password; labelColor: black"
```

Comment:
What does `hand: #mouseCursor` do?

Reply:
Yeah I'm also not sure what this does. But it looks like you removed it so I guess it's not needed? Can you confirmed Amolik

Reply:
![image](https://user-images.githubusercontent.com/25352114/112733170-b4c72500-8f14-11eb-9f5e-bd086a6027e8.png)